test/lib/model/test_yake_keywords.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from unittest.mock import ANY, MagicMock, patch
+
+from lib.model.yake_keywords import Model
+from lib import schemas
+
+class TestYakeKeywordsModel(unittest.TestCase):
+    def setUp(self):
+        self.yake_model = Model()
+
+    @patch('yake.KeywordExtractor.extract_keywords')
+    def test_process(self, mock_yake_response):
+        message = schemas.Message(**{
+            "body": {
+                "id": "1234",
+                "text": "Some Text",
+            },
+            "model_name": "yake_keywords__Model"
+        })
+        mock_yake_response.return_value = [["ball", 0.23]]
+        self.assertEqual(self.yake_model.process(message), {"keywords": [["ball", 0.23]]})
+
+    @patch('yake.KeywordExtractor.extract_keywords')
```

Comment:
This looks too me like we are patching the return value and not actually calling the model that we are trying to test?